crm_lead_firstname/__manifest__.py

```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Firstname and Lastname in Leads",
     "summary": "Specify split names for contacts in leads",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Customer Relationship Management",
     "website": "http://www.github.com/OCA/crm",
     "author": "Antiun Ingeniería S.L., Odoo Community Association (OCA)",
```

Comment:
Please update to the new company `Tecnativa`.

Reply:
What should be updated can you insert a code suggestion?
I don't know what I need to change...

Reply:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```